composer/callbacks/global_straggler_detector.py

```diff
@@ -0,0 +1,873 @@
+from megatron.core.utils import *
```

Comment:
Can we just import the functions that we need from utils so that way we can remove our dependency from megatron?